test/13-unit-tests-base.tpkg/13-unit-tests-base.c

```diff
@@ -5,6 +5,10 @@
 
 #include <ldns/ldns.h>
 
+/* Avoid signedness warnings */
+#define CH_PTR(ptr) ((char*)(ptr))
+#define UCH_PTR(ptr) ((unsigned char*)(ptr))
```

Comment:
Hi @noloader , On hindsight I think I would prefer direct type conversions instead of via a macro. I'll do the change myself. But just so you know. Thanks anyway for all the fixes and features! Very much appreciated.